pressbooks/pressbooks-multi-institution
# feat: add institutional manager stats

Issue: https://github.com/pressbooks/pressbooks-multi-institution/issues/67

This PR hooks into some PB Network Analytics filters added in https://github.com/pressbooks/pressbooks-network-analytics/pull/343 and sends the new institution column, it also joins the institution-related tables and adds conditions to filter the NM stats, it also adds the institution column into some download files, modifies the filename for CSV files and finally it modifies the main title of the page.

### Testing case
For testing it, you will need to make sure you have the changes proposed in: https://github.com/pressbooks/pressbooks-network-analytics/pull/343

As a Super Admin and NM:
- Log in as a Super Admin or NM.
- Go to the Stats page.
- You should see the new title on the page: `Network Stats`.
- Check the stats and download CSV files. Everything should work just as usual.

As an IM:
- Log in as an IM.
- Go to the Stats page.
- You should see the new title on the page: `<Institution name> Stats`.
- All the data you can see and download through CSV should be only related to the institution associated to the user.
- Any file you download from there, should contain the institution name.
- When you download Page view, Network storage, Most cloned books and User revision over last CSV files, you should see a new `Institution` column populated.
